### PR TITLE
Add Polyfil function support

### LINKED ISSUE #1650: [PHP] for new version code verify both version and polyfil
E.g. `is_iterable` can be added via symfony polyfil before PHP version is upgraded

Suggested by @stloyd

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #1650

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Cc @@stloyd

File: packages/Php/src/Rector/BinaryOp/IsCountableRector.php [hunk 1]
```diff
@@ -52,10 +52,19 @@ public function getNodeTypes(): array
...
             return null;
         }
 
-        return $this->dualCheckToAble->processBooleanOr($node, 'Countable', 'is_countable') ?: $node;
+        return $this->isArrayAndDualCheckToAble->processBooleanOr($node, 'Countable', 'is_countable') ?: $node;
+    }
+
+    private function shouldSkip(): bool
+    {
+        if ($this->isAtLeastPhpVersion('7.3')) {
```
Comment (stloyd) [inline packages/Php/src/Rector/BinaryOp/IsCountableRector.php]: Shouldn't those check be opposite?

You can have polyfill with that method on even PHP 7.0, so first it should check for polyfill/method availability, then check PHP version, no?

Comment (TomasVotruba) [reply packages/Php/src/Rector/BinaryOp/IsCountableRector.php]: You're right of course!

Care for you PR?

Comment (stloyd) [reply packages/Php/src/Rector/BinaryOp/IsCountableRector.php]: https://github.com/rectorphp/rector/pull/1677

**Pull request merged**